### PR TITLE
Adding a Footer and modifying Share component

### PR DESCRIPTION
**Pull request by rickiesherman**
Comparison shots 

New (Full size)
---
screen shot 2017-02-01 at 11 36 27 am

Old (Full size)
---
screen shot 2017-02-01 at 11 36 22 am

Half size (Left is old, Right is new)
---
screen shot 2017-02-01 at 11 36 03 am

### CODE REVIEW

**Comment by tosih** [comment]
Looks a little funky at the bottom
image

**Comment by rickiesherman** [comment]
@tosih i'll try to fix that, good looks

**Review by tosih — CHANGES_REQUESTED** [review]
Looks a little funky at the bottom

image

Basically the alignment of the footer objects conflicts with the sharing links.

**Comment by runnabro** [comment]
Can we also add the other footer elements (the support widget + the sign up CTA)?

File: _includes/footer.html [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
-  <div class="grid-block">
-    <a href="{{ site.url }}/about/" class="btn btn-xs link strong">About</a>
-    <a href="{{ site.url }}/privacy/" class="btn btn-xs link strong">Privacy Policy</a>
+<footer class="grid-block col footer justify-spaced">
+  <div class="grid-block link-wrapper">
+    <div class="grid-block col align-start">
+      <a href="{{ site.url }}/for/" class="btn btn-xs link weight-strong">Product</a>
+      <a href="{{ site.url }}/pricing/" class="btn btn-xs link weight-strong">Pricing</a>
+      <a href="{{ site.url }}/how-it-works/" class="btn btn-xs link weight-strong">How It Works</a>
+      <a href="https://support.runnable.com" class="btn btn-xs link weight-strong">Get Help</a>
```
Comment (runnabro) [inline _includes/footer.html]: This link has been removed from our other footers.

File: _includes/footer.html [hunk 2]
```diff
@@ -1,6 +1,20 @@
-<footer class="grid-block vertical footer justify-spaced">
-  <div class="grid-block">
-    <a href="{{ site.url }}/about/" class="btn btn-xs link strong">About</a>
-    <a href="{{ site.url }}/privacy/" class="btn btn-xs link strong">Privacy Policy</a>
+<footer class="grid-block col footer justify-spaced">
+  <div class="grid-block link-wrapper">
+    <div class="grid-block col align-start">
+      <a href="{{ site.url }}/for/" class="btn btn-xs link weight-strong">Product</a>
```
Comment (runnabro) [inline _includes/footer.html]: We added a new docs link here.

File: _sass/components/share.scss [hunk 3]
```diff
@@ -1,6 +1,8 @@
 .share {
   border-top: 1px solid $gray-lighter;
-  padding: $md $md $md 0;
+  margin: 0 auto;
+  text-align: center;
+  padding: $md 0 $md 0;
```
Comment (runnabro) [inline _sass/components/share.scss]: This needs to be in alphabetical order

File: _sass/components/share.scss [hunk 4]
```diff
@@ -10,11 +12,18 @@
   .btn + .btn {
     margin-left: $xs;
 
-    @include media(xs) {
+    @include media(md) {
       margin-left: 0;
       margin-top: $xxs;
     }
   }
+  .btn-sm {
```
Comment (runnabro) [inline _sass/components/share.scss]: Should have an empty space between selectors.

File: _sass/components/share.scss [hunk 5]
```diff
@@ -10,11 +12,18 @@
...
+    @include media(md) {
+      width: 100%;
+      max-width: 480px;
+      margin: 0 auto;
```
Comment (runnabro) [inline _sass/components/share.scss]: This needs to be in alphabetical order

File: _sass/landing/footer.scss [hunk 6]
```diff
@@ -22,7 +24,9 @@
   }
 
   .link-wrapper {
-    padding: 60px;
+    margin: 0 auto;
+    padding: 0;
+    max-width: 520px;
```
Comment (runnabro) [inline _sass/landing/footer.scss]: Ordering

**Review by tosih — APPROVED** [review]

**Comment by tosih** [comment]
Wait, it still looks a little off.

**Comment by tosih** [comment]
Is this desired? @taylordolan @taylordolan @rickiesherman 
image

**Comment by rickiesherman** [comment]
@tosih nah, I'll have a look at it

**Comment by rickiesherman** [comment]
This is what it's looking like right now: 
screen shot 2017-02-07 at 9 33 15 am

**Comment by tosih** [comment]
LGTM

**Comment by taylordolan** [comment]
image
at 640px

**Review by taylordolan — APPROVED** [review]

**Pull request merged**